tools/cbmc/proofs/CBMCStubLibrary/tasksStubs.c

```diff
@@ -0,0 +1,46 @@
+#include "FreeRTOS.h"
+#include "task.h"
+#include "tasksStubs.h"
+
+#ifndef TASK_STUB_COUNTER
+	#define TASK_STUB_COUNTER 0;
+#endif
+
+/* 5 is a magic number, but we need some number here as a default value.
+   This value is used to bound any loop depending on xTaskCheckForTimeOut
+   as a loop bound. */
+#ifndef TASK_STUB_COUNTER_LIMIT
+	#define TASK_STUB_COUNTER_LIMIT 5;
```

Comment:
Could you justify 5 by saying the verification time goes to the roof if you use 10 for instance? (justifying by performance)